### DISCUSSION TITLE
Wine-devel is no longer tracking updates

### DISCUSSION THREAD

**Discussion by w1ebr** [Homebrew/discussions#5298] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

wine-devel stopped tracking updates at version 9.3. Current version is 9.6

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by w1ebr** [comment]
I am not skilled enough at this time to do that

**Comment by daeho-ro** [comment]
I opened a PR for this.
https://github.com/Homebrew/homebrew-cask-versions/pull/20095

It seems that the mac version of wine is uploaded within 20 minutes.
https://github.com/Gcenx/macOS_Wine_builds/releases

**Comment by w1ebr** [comment]
Thank you so much!

**Comment by w1ebr** [comment]
It isn't running yet. This is what I get when I run the program
`gene@Genes-MBP ~ % wine ~/Desktop/DSDPlus1p101/DSDPlus.exe
dyld[15925]: Library not loaded: /opt/local/lib/libinotify.0.dylib
  Referenced from: <D8F464E3-D15B-3DB9-9D3A-216C4405B7C2> /Applications/Wine Devel.app/Contents/Resources/wine/bin/wineserver
  Reason: tried: '/opt/local/lib/libinotify.0.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/local/lib/libinotify.0.dylib' (no such file), '/opt/local/lib/libinotify.0.dylib' (no such file)`

The dylibs are in the Resources directory in Wine Development.app. I could manually copy the files from the Resources director into "opt/local/lib" but that seems like a bad idea.

Any suggestions of what I should do next? Thanks for all of the help!
